Use of unsigned char instead of char

xseignard opened the pull request:
In order to display non ascii chars, we need to be able to go outside the range of -128/127 the char type handles.

Using `unsigned char` let us handle extend ascii chars till 255.

projectgus committed:
Add 'const' to PROGMEM font declarations

xseignard committed:
Use of unsigned char instead of char

In order to display non ascii chars, we need to be able to go outside the range of -128/127 the char type handles.

Using `unsigned char` let us handle extend ascii chars till 255.

xseignard committed:
Update DMD.h

xseignard committed:
Removing unecessary cast

xseignard committed:
Added a font with non ascii chars

xseignard committed:
Added a font with non ascii chars

xseignard committed:
Removed font with wrong name

xseignard committed:
Merge branch 'patch-1' of github.com:xseignard/DMD into patch-1

projectgus commented:
Thanks very much for this Xavier! I think it will be very popular.

One minor request, could you please add a comment to Arial_Black_16_Extended.h describing which characters are present in the font? Is it the full ISO/IEC 8859-1 "Latin-1" set, or is it a smaller range?

Really appreciate this, though. Several people have asked about extended character support, it's great to be able to supply a font that works with it.

xseignard committed:
Better explenation of the added font

- Renamed it to an explicit name
- Added comment on the font mentionning the charset.
